lib/mods/replace-get-set.js

```diff
@@ -48,7 +48,8 @@ function isDataPropertyAccess(node) {
  */
 function isGlobalPropertyAccess(node) {
   return nodeTypes.isPropertyAccessExpression(node) &&
-    !nodeTypes.isCallExpression(node.getParent()) &&
+    !(nodeTypes.isCallExpression(node.getParent()) &&
+    node.getParent().getExpression() === node) &&
```

Comment:
Is it possible to express what this condition is asking? e.g.

```js
const ___something___ = nodeTYpes.isCallExpression(node.getParent()) && node.getParent().getExpression === node;
```

and then just use `!___something__`? Where `__something__` is a semantic name indicating what actually you're testing for...

Reply:
I was considering documenting via comments but this seems like a good way to do it, I'll change to this

Reply:
Updated to a separate var with a explanatory comment